src/main/java/com/google/flourbot/entity/action/sheet/SheetReadColumnAction.java

```diff
@@ -0,0 +1,29 @@
+package com.google.flourbot.entity.action.sheet;
+
+import com.google.flourbot.api.DriveCloudSheet;
+import com.google.flourbot.entity.action.ActionType;
+
+public final class SheetReadColumnAction extends SheetAction {
+
+  private final ActionType actionType = ActionType.SHEET_READ_COLUMN;
+  private final String column;
+
+  public SheetReadColumnAction(String sheetUrl, String column) {
+    super(sheetUrl);
+    this.column = column;
+  }
+
+  public SheetReadColumnAction(String sheetUrl, int column) {
```

Comment:
When would we use a string vs an int?

Reply:
This was written to make sure it accepts both formats, but actually yeah, the int part isn't as important if we can make sure a String is what we have in Firestore. Currently, other code in entitymoduleimplementation assumes that it's always a string, but soon I hope we can add an option in the UI where a user can supply either one.

Reply:
Ok! I like giving users the options, my only concern is will this mean we will have null values in the database? Will this be a problem?

Reply:
Oh if they key isn't there, it returns null, and we can handle that! So we won't have nulls in the firestore